app/api/stripe/webhook/route.ts

```
@@ -87,6 +87,20 @@ export async function POST(request: Request) {
         // The payment failed or the customer does not have a valid payment method.
         // The subscription becomes past_due. Notify your customer and send them to the
         // customer portal to update their payment information.
+        // Also update the subscription status in our database
+        const failedInvoice = receivedEvent.data.object as Stripe.Invoice;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)